tests/fast-integration/package.json

```diff
@@ -1,6 +1,14 @@
 {
-  "name": "fast-integration",
-  "version": "0.0.1",
+  "name": "kyma-js",
```

Comment:
I'm a bit confused as we are mixing here (in this package) two things. The standalone kyma installation CLI and the tests that covers more thant the CLI responsibility.

Reply:
I was thinking about it. If we should separate the tool. But I treat it as a helper to run the integration tests locally. Long term kyma-cli should cover all the commands here. But it is not there yet. Keeping everything together helps a lot too speed up development. See the example: https://terminalizer.com/view/f55226d14700

Reply:
I added disclaimer to the overview in the README to point out that kyma-js is just a set of developers' tools, not the official CLI.